### ISSUE TITLE
[CI] MixedClusterClientYamlTestSuiteIT test {p0=bulk/60_deprecated/Deprecated parameters should produce warning in Bulk query}

### ISSUE THREAD

**Issue by ywangd** [elastic/elasticsearch#54545]
Edit: This started to fail again since 6.8.8 was created and seems to related #53923. Probably need to disable it for 6.8.8. Asking @henningandersen to take a look.

Branch 6.8: The failure can be reproduced locally with
```
./gradlew ':qa:mixed-cluster:v6.8.8#mixedClusterTestRunner' \
  -Dtests.seed=D0EAA82870892007 \
  -Dtests.class=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT \
  -Dtests.method="test {p0=bulk/60_deprecated/Deprecated parameters should produce warning in Bulk query}" \
  -Dtests.security.manager=true \
  -Dtests.locale=de-GR \
  -Dtests.timezone=America/Denver \
  -Dtests.distribution=default \
  -Dcompiler.java=12 \
  -Druntime.java=8
```

Build Job:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+default-distro+bwc/BWC_VERSION=6.8.8,nodes=centos-7&&immutable/714/console

Failure email:
https://groups.google.com/a/elastic.co/forum/?utm_medium=email&utm_source=footer#!msg/build-elasticsearch/iZs1dswONuQ/Zj74fQrbAwAJ

Build scan:
https://gradle-enterprise.elastic.co/s/k35eh57sjole6

Somehow it is difficult to locate the actual error message in the build scan. It is pasted here:

```
java.lang.AssertionError: Failure at [bulk/60_deprecated:10]: got unexpected warning headers [
	299 Elasticsearch-6.8.8-2f4c224 "Usage of internal versioning for optimistic concurrency control is deprecated and will be removed. Please use the `if_seq_no` and `if_primary_term` parameters instead. (request for index [test_index], type [test_type], id [test_id_1])"
	299 Elasticsearch-6.8.8-2f4c224 "Usage of internal versioning for optimistic concurrency control is deprecated and will be removed. Please use the `if_seq_no` and `if_primary_term` parameters instead. (request for index [test_index], type [test_type], id [test_id_2])"
]
did not get expected warning header [
	Usage of internal versioning for optimistic concurrency control is deprecated and will be removed. Please use the `if_seq_no` and `if_primary_term` parameters instead. (request for index [test_index], type [test_type])
]

```

**Comment by ywangd** [comment]
This started to fail since 6.8.8 is created. Three failures so far. Before these, similar failures were fixed by [skipping these tests up to version 6.8.7](https://github.com/elastic/elasticsearch/pull/53923).

@henningandersen Could you please check whether the skipping versions should be incremented to 6.8.8?

**Comment by henningandersen** [comment]
Thanks @ywangd , I fixed this in #54560.
